core/conventional-commits/index.js

```diff
@@ -119,12 +119,31 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath }) {
         }
 
         log.verbose(type, "increment %s by %s", pkg.version, data.releaseType);
-        resolve(semver.inc(pkg.version, data.releaseType));
+        resolve({
+          currentVersion: pkg.version,
+          nextVersion: semver.inc(pkg.version, data.releaseType),
+        });
       });
     });
   });
 }
 
+function isBreakingChange(currentVersion, nextVersion) {
```

Comment:
This doesn't belong here.

Reply:
I tried to find a place to put it, any suggestion?